colossalai/shardformer/policies/llama.py

```diff
@@ -377,6 +410,20 @@ def get_shared_params(self) -> List[Dict[int, Tensor]]:
                         self.pipeline_stage_manager.num_stages - 1: self.model.lm_head.weight,
                     }
                 ]
+                # if self.pipeline_stage_manager.use_zbv:
```

Comment:
why comment

Reply:
> why comment

Removed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/fafe049b83bad3a6aa6e3a31c68b38ac63167b53;